src/App.js

```diff
@@ -1,19 +1,28 @@
 import React from "react";
 import { Route, Routes } from "react-router-dom";
+import { useState } from "react";
 import ReviewList from "./Componenets/ReviewList";
 import Header from "./Componenets/Header";
 import NavBar from "./Componenets/navBar";
 import Categories from "./Componenets/Categories";
 import SingleReview from "./Componenets/SingleReview";
 import Comments from "./Componenets/Comments";
 const App = () => {
+  const [selectedCategory, setSelectedCategory] = useState("");
+
+  const handleCategorySelection = (selectedCategory) => {
```

Comment:
This function seems to do the same thing as `setSelectedCategory`, so `setSelectedCategory` should really be what you're sending into the Categories component. Then you can remove this `handleCategorySelection` from this file.